android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -286,10 +268,7 @@ public List<CryptoAccountTypeInfo> getSupportedCryptoAccountTypes() {
 
         @Override
         public List<Integer> getSupportedCryptoCoins() {
-            return getSupportedCryptoAccountTypes()
-                    .stream()
-                    .map(CryptoAccountTypeInfo::getCoinType)
-                    .collect(Collectors.toList());
+            return Arrays.asList(CoinType.ETH, CoinType.SOL, CoinType.FIL, CoinType.BTC);
```

Comment:
Technically there is a feature flag for BTC which might be disabled, so coin should not be seen as supported.

Reply:
I would also add that getSupportedCryptoCoins() is only used in NetworkModel to fetch all available networks from core which itself should be refactored to be just getAllNetworks() - not per coin, so getSupportedCryptoCoins would not be needed at all

Reply:
Yes! It's something I would like to remove soon, but the PR was already fairly big, so to keep scope smaller I've created a follow-up issue for this 👉 https://github.com/brave/brave-browser/issues/37398